content/getting-started/kotlin.textile

```diff
@@ -0,0 +1,183 @@
+
+h2(#prerequisites). Prerequisites
+
+* Sign up for an Ably account.
+** Create a new app, and create your first API key.
+** Your API key will need the @publish@, @subscribe@, @presence@ and @history@ capabilities.
+
+* Install the Ably CLI:
+
+```[sh]
+npm install -g @ably/cli
+```
+
+* Run the following to log in to your Ably account and set the default app and API key:
+
+```[sh]
+ably login
+
+ably apps switch
+ably auth keys switch
+```
+
+* Install "JDK":https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html version 11 or greater.
+* Create a new project in your IDE and install the Ably Pub/Sub Kotlin SDK. If using Gradle:
+
+```[kotlin]
+implementation("io.ably:ably-java:<latest-version>")
+```
+
+<aside data-type='note'>
+<p>The code examples in this guide include a demo API key. If you wish to interact with the Ably CLI and view outputs within your Ably account, ensure that you replace them with your own API key.</p>
+</aside>
+
+h2(#step-1). Step 1: Connect to Ably
+
+Clients establish a connection with Ably when they instantiate an SDK. This enables them to send and receive messages in realtime across channels.
+
+* Open up the "dev console":https://ably.com/accounts/any/apps/any/console of your first app before instantiating your client so that you can see what happens.
+
+* Create an @Main.kt@ file in your project and add the following function to instantiate the SDK and establish a connection to Ably. At the minimum you need to provide an authentication mechanism. Use an API key for simplicity, but you should use token authentication in a production app. A @clientId@ ensures the client is identified, which is required to use certain features, such as presence:
+
+
+```[kotlin]
+import io.ably.lib.realtime.AblyRealtime
+import io.ably.lib.realtime.ConnectionEvent
+import io.ably.lib.types.ClientOptions
+
+fun getStarted() {
+  val realtimeClient = AblyRealtime(ClientOptions().apply {
+    key = "{{API_KEY}}"
+    clientId = "my-first-client"
+  })
+
+  realtimeClient.connection.once(ConnectionEvent.connected) {
+    println("Made my first connection!")
+  }
+}
+
+fun main() {
+  getStarted()
+}
+```
+
+You can monitor the lifecycle of clients' connections, but for now just log a message to the console to know that the connection attempt was successful. You'll see the message printed to your console, and you can also inspect the connection event in the dev console of your app.
+
+h2(#step-2). Step 2: Subscribe to a channel and publish a message
+
+Messages contain the data that a client is communicating, such as a short 'hello' from a colleague, or a financial update being broadcast to subscribers from a server. Ably uses channels to separate messages into different topics, so that clients only ever receive messages on the channels they are subscribed to.
+
+* Add the following lines to your @getStarted()@ function to create a channel instance and register a listener to subscribe to the channel. Then run it:
```

Comment:
Same here - we need to update all instances of  "Then run it with..." please to include the 'how'.